gsoc/models.py

```diff
@@ -3,6 +3,23 @@
 from django.contrib.auth.models import User
 
 
+class Scheduler(models.Model):
+    commands = (
+        ('send_email', 'send_email'),
+        ('send_irc_msg', 'send_irc_msg')
+    )
+    
+    id = models.AutoField(primary_key=True)
+    command = models.CharField(name='command', max_length=20, choices=commands)
+    data = models.TextField(name='data')
+    success = models.BooleanField(name='success', null=True, default=None)
+    last_error = models.CharField(name='last_error', max_length=100, null=True, default=None)
```

Comment:
A max_length=100 char field for errors may be quite small if there is like a stack trace or something especially. TextField probably the better choice.

Reply:
Yeah, even I was thinking about this. I'll change this.